src/components/BorrowerProfile/LendersAndTeams.vue

```diff
@@ -142,11 +142,11 @@ import KvMaterialIcon from '~/@kiva/kv-components/vue/KvMaterialIcon';
 import KvTextLink from '~/@kiva/kv-components/vue/KvTextLink';
 import SupporterDetails from './SupporterDetails';
 
-const teamsQuery = gql`query teamsQuery($loanId: Int!, $limit: Int, $offset: Int) {
+const teamsQuery = gql`query teamsQuery($loanId: Int!, $limit: Int, $offset: Int, $sortBy: String) {
```

Comment:
Oh actually, you should be able to swap out `String` for `TeamSearchSortByEnum` to clear this warning too.